docs/Configuration.md

```diff
@@ -70,8 +70,16 @@ countsSettings {
   password: "admin"
 }
 
-```
-In the next sections we'll look at 
+```             
+###Default configurations
+Default configuration for UI is in [defaultConfig.conf](https://github.com/TouK/nussknacker/blob/staging/ui/server/src/main/resources/defaultConfig.conf).
```

Comment:
It would be nice to have some more clarified distinguishion of default configs for UI and for model. Maybe we should name it `defaultUiConfig.conf`?

Reply:
ok, renamed